Resources/doc/reference/architecture.rst

```diff
@@ -356,5 +356,26 @@ Then, you have to set the CommentAdmin ``parentAssociationMapping`` attribute to
 
 It also possible to set a dot-separated value, like ``post.author``, if your parent and child admins are not directly related.
 
+Be wary that being a child admin is optional, which means that regular routes
+will be created regardless of whether you actually need them or not. To get rid
+of them, you may override the `configureRoute` method::
+
+    <?php
```

Comment:
Missing `.. code-block:: php` before the block.

BTW, is the `<?php` opening tag really necessary?